user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md

```diff
@@ -48,6 +48,11 @@ Especially the name and state of the index may be interesting for debugging purp
 
 ### Manually setting the replication count for indices
 
+Follosing commands allow you to alter the number of replicas (per shard) for indices matching the specific criteria. This is useful if you would like to reduce your per-node shard count and also if there are more replicas then the number of nodes, which is not very useful and put an unnecessary strain to the system. Furthermore you can use it to cutomize the availability of your ElasticSearch database.
```

Comment:
Several spelling mistakes. Would reword this entire paragraph e.g. as such: 

The following commands allow you to alter the number of replicas per shard. 
This would typically be used to configure redundancy and availability based on the amount of Elasticsearch nodes you have available.

!TIP 
If the amount of shards you have is too large, decreasing the amount of replicas per shard is one option to decrease the strain on the system.